front/lib/api/assistant/conversation.ts

```diff
@@ -413,6 +421,150 @@ export async function getConversation(
   });
 }
 
+export async function getPokeConversation(
```

Comment:
Any place to put it that is more poke specific?

Reply:
Afaik, other Poke methods are next to the regular methods.

Reply:
Yep I think it's def time to cut a lib/api/poke.ts as a starter?